pulsar-io/kafka-connect-adaptor/pom.xml

```diff
@@ -116,6 +116,12 @@
       <groupId>io.confluent</groupId>
       <artifactId>kafka-connect-avro-converter</artifactId>
       <version>${confluent.version}</version>
+      <exclusions>
```

Comment:
Why need to exclude this?

Reply:
Otherwise it pulls in avro 1.10.3 which is suffered from the CVE.

And actually, we have a provided avro dep in this pom.xml file so I believe we forget to exclude it here.

Reply:
Got it, thanks.